mlfinlab/labeling/trend_scanning.py

```diff
@@ -5,8 +5,53 @@
 
 import pandas as pd
 import numpy as np
+from numba import njit
 
-from mlfinlab.structural_breaks.sadf import get_betas
+
+@njit
+def calculate_t_values(subset, min_sample_length, step):  # pragma: no cover
+    """
+    For loop for calculating linear regression every n steps.
+
+    :param subset: (np.array) subset of indecies for which we want to calculate t values
+    :return: (float) maximum t value and index of maximum t value
```

Comment:
Please, add a description for the `min_sample_length` variable.

Also, please start variables descriptions with a capital letter.